Misc/NEWS.d/next/Library/2020-09-08-23-41-29.bpo-41747.M6wLKv.rst

```diff
@@ -0,0 +1,3 @@
+Ensure all methods belonging to :func:`dataclasses.dataclass` objects
```

Comment:
This should somehow say that it only applies to methods that dataclasses add, not all methods.